semgrep-core/src/engine/Match_tainting_rules.mli

```diff
@@ -5,3 +5,25 @@ val check_rule :
   Rule.taint_spec ->
   Xtarget.t ->
   Report.rule_profiling Report.match_result
+
+(* Deep Semgrep *)
```

Comment:
Can this be moved in the other repo?

Reply:
Good point, I think that it can be removed from here indeed. I'll try doing that.